src/index.test.js

```diff
@@ -200,20 +184,20 @@ describe('cmp.showPrivacyManager', () => {
 		cmp.showPrivacyManager();
 
 		return waitForExpect(() =>
-			expect(TCFv2.showPrivacyManager).toHaveBeenCalledTimes(1),
+			expect(CMP.showPrivacyManager).toHaveBeenCalledTimes(1),
 		);
 	});
 });
 
 describe('Old API parameter `isInUsa`', () => {
```

Comment:
This `isInUsa` param should be removed. It’s been deprecated since [6.5](https://github.com/guardian/consent-management-platform/blob/v6.5.0/src/index.ts).

Reply:
Good call - I've started [another branch + pr](https://github.com/guardian/consent-management-platform/pull/498) in the interest of keeping this one as light at possible.